### PR TITLE
specify user/project domain by name

### LINKED ISSUE #1: allow user and project from different domain
The `domain_id` configuration option is used [both to locate the user and the project](https://github.com/FujitsuEnablingSoftwareTechnologyGmbH/logstash-output-monasca_log_api/blob/e9214b1c8d1ae6d75e33bd21c0f3579678b8f0a6/lib/logstash/outputs/keystone/keystone_client.rb#L61-L82). However, it is possible that user and project are in different domains, in which case two different domain IDs would need to be specified.

### PR DESCRIPTION
**Pull request by majewsky**
This does two things:

1. change the `domain_id` to a domain name parameter, since names are easier to handle and don't change in unexpected ways during maintenances as often as IDs do

2. fix #1 by splitting said parameter into two, `user_domain_name` and `project_domain_name`

If you like this, please merge this into your master so it shows up in [your upstreaming PR](https://github.com/logstash-plugins/logstash-output-monasca_log_api/pull/1).

~~WIP: I forgot about the specs. Will adjust these and make them green. But you can have a look at the code changes already.~~ Done.

### CODE REVIEW

**Comment by witekest** [comment]
Thanks a lot for your contribution! We'll review that.

File: README.md [hunk 1]
```diff
@@ -128,7 +128,8 @@ Save the configfile wherever you like. For example ~/logstash.conf
 | project_name | Keystone user credentials: project name | string | true | | monasca |
 | username | Keystone user credentials: username | string | true | | admin-agent |
 | password | Keystone user credentials: password | string | true | | password |
-| domain_id | Keystone user credentials: domain-id | string | true | | default |
+| user_domain_name | Keystone user credentials: domain-id | string | true | | default |
```
Comment (kejc87) [inline README.md]: we should write in the description user domain-id and for project project domain-id

**Comment by kejc87** [comment]
Thanks for your contribution. Code looks so far good. Could you please also upgrade the version number from 0.5 to 0.5.1.

**Comment by majewsky** [comment]
@kejc87 Thanks for the hint, I adjusted the version number and adjusted the wording in the documentation.

**Comment by kejc87** [comment]
Sorry for waiting (I was in vacation) and thank you for your contribution!

**Pull request merged**